src/SprykerEco/Yves/Braintree/Handler/BraintreeHandler.php

```diff
@@ -111,13 +111,13 @@ protected function setBraintreePayment(Request $request, QuoteTransfer $quoteTra
             ->setCurrencyIso3Code($this->getCurrency())
             ->setLanguageIso2Code($billingAddress->getIso2Code())
             ->setClientIp($request->getClientIp())
-            ->setNonce($nonce);
+            ->setNonce((string)$nonce);
```

Comment:
there is a default value parameter present at Request::request
use ot instead of casting types

Reply:
Casting is required anyway because `$request->request->get()` return type is `string|int|float|bool|null`. I moved the casting from this line to the result of the  `$request->request->get()` call itself. Now the type of `$nonce` is always string, which looks a bit clearer (although the casting is still there)